### PR TITLE
Add event

### PR DESCRIPTION
**Pull request by shriharsha05**

### CODE REVIEW

File: src/views/addEvent.vue [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            <b-field label="Title">
+                <b-input v-model="title"></b-input>
+            </b-field>
+
+            <b-field label="Image Link">
+                <b-input v-model="link"></b-input>
+            </b-field>
+
+            <b-field label="Description">
+                <b-input maxlength="200" type="textarea" vmodel="desc"></b-input>
```
Comment (mridulganga) [inline src/views/addEvent.vue]: vmodel should be v-model

File: src/views/addEvent.vue [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  data: function() {
+    return {
+      submitted: false
+    };
+  },
+
+  methods: {
+    // submit form handler
+    submit: function() {
+    this.submitted = true;
```
Comment (mridulganga) [inline src/views/addEvent.vue]: what is this.submitted for?

File: src/views/addEvent.vue [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    var day = today.getDate() + "";
+    var month = (today.getMonth() + 1) + "";
+    var year = today.getFullYear() + "";
+    var hour = today.getHours() + "";
+    var minutes = today.getMinutes() + "";
+    var seconds = today.getSeconds() + "";
+
+    var timestamp = day + "/" + month + "/" + year + " " + hour + ":" + minutes + ":" + seconds;
+
+    var params = {   "title": this.title,
```
Comment (mridulganga) [inline src/views/addEvent.vue]: might need to add some validation later. You may resolve this right now.

File: src/views/addEvent.vue [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+                     "venue" : this.venue,
+                     "timestamp" : timestamp,
+                     "fee" : {
+                         "ieee" : this.ieee_fee,
+                         "non" : this.non_ieee_fee,
+                         "cs" : this.cs_fee
+                     } 
+    };
+
+    console.log(params);
```
Comment (mridulganga) [inline src/views/addEvent.vue]: remove console.log after testing it.

File: src/views/addEvent.vue [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+                     "timestamp" : timestamp,
+                     "fee" : {
+                         "ieee" : this.ieee_fee,
+                         "non" : this.non_ieee_fee,
+                         "cs" : this.cs_fee
+                     } 
+    };
+
+    console.log(params); 
+    axios.post('http://localhost:8080/#/event', params);
```
Comment (mridulganga) [inline src/views/addEvent.vue]: add a `.then(Response=>{   })` and validate that you got correct response.

File: src/router.js [hunk 6]
```diff
@@ -24,6 +25,11 @@ export default new Router({
       name: 'notices',
       component: Notices
     },
+    {
+      path: '/event/add',
+      name: 'addEvent',
+      component: addEvent
```
Comment (mridulganga) [inline src/router.js]: See if we should rename the component to AddEvent

File: src/views/addEvent.vue [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+  data: function() {
+    return {
+      submitted: false
+    };
+  },
+
+  methods: {
+    // submit form handler
+    submit: function() {
```
Comment (mridulganga) [inline src/views/addEvent.vue]: this wont get invoked on its own. you need to add a @click handler in the submit button.
Maybe we can rename this to addEvent

**Review by mridulganga — CHANGES_REQUESTED** [review]

File: src/views/addEvent.vue (hunk 1 above)
Comment (shriharsha05) [reply src/views/addEvent.vue]: how silly of me :/

File: src/views/addEvent.vue (hunk 2 above)
Comment (shriharsha05) [reply src/views/addEvent.vue]: removing.

File: src/views/addEvent.vue (hunk 3 above)
Comment (shriharsha05) [reply src/views/addEvent.vue]: Understood :)

File: src/views/addEvent.vue (hunk 4 above)
Comment (shriharsha05) [reply src/views/addEvent.vue]: sure.

File: src/router.js (hunk 6 above)
Comment (shriharsha05) [reply src/router.js]: Renamed.

File: src/views/addEvent.vue (hunk 7 above)
Comment (shriharsha05) [reply src/views/addEvent.vue]: Noted and Added @click

**Review by shriharsha05 — COMMENTED** [review]
Made necessary changes.

**Review by mridulganga — APPROVED** [review]

**Pull request merged**